### PR TITLE
Tidy spatial Subset info on Step 3

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

File: src/test/javascript/portal/prototypes/OpenLayersSpec.js [hunk 1]
```diff
@@ -177,6 +177,12 @@ describe('OpenLayers', function() {
                 expect(OpenLayers.Geometry.fromWKT('POLYGON((1 2,3 4,1 2))').isBox()).toEqual(false);
             });
         });
+
+        describe('getPrettyBounds', function() {
+            it('returns true when box', function() {
```
Comment (dnahodil) [inline src/test/javascript/portal/prototypes/OpenLayersSpec.js]: This message does not match what we're actually testing

File: src/test/javascript/portal/prototypes/OpenLayersSpec.js [hunk 2]
```diff
@@ -177,6 +177,12 @@ describe('OpenLayers', function() {
...
+                expect(OpenLayers.Geometry.fromWKT('POLYGON((1 2,3 2,3 4,1 4,1 2))').getPrettyBounds()).toEqual('1W 2S 3E 4N');
```
Comment (dnahodil) [inline src/test/javascript/portal/prototypes/OpenLayersSpec.js]: We should probably test with values with decimal components to ensure that toNSigFigs() is used.

**Comment by dnahodil** [comment]
The code changes here look really good @pmbohm, and this will be a nice change to have.

I've just made 2 comments.

**Comment by pmbohm** [comment]
@dnahodil tests now changed

**Comment by dnahodil** [comment]
:+1:

**Pull request merged**